Cargo.toml

```diff
@@ -1,29 +1,29 @@
 [package]
 name = "quilt-installer"
 description = "The installer for quilt-loader"
-
 version = "0.1.1"
 license = "Apache-2.0"
-
 edition = "2021"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
-anyhow = "1.0"
-async-trait = "0.1"
-base64 = "0.13"
-chrono = "0.4"
-clap = { version = "3.2", features = ["derive"] }
-iced = { version = "0.3", default-features = false, features = ["glow", "tokio"] }
-iced_glow = "0.2"
-native-dialog = "0.6.3"
-png = "0.17.5"
 reqwest = { version = "0.11", features = ["blocking", "json"] }
+iced = { version = "0.6", default-features = false, features = [
+    "glow",
+    "tokio",
+] }
+chrono = { version = "0.4", features = ["serde"] }
+semver = { version = "1.0", features = ["serde"] }
 serde = { version = "1.0", features = ["derive"] }
+clap = { version = "4.0", features = ["derive"] }
+native-dialog = "0.6"
+dark-light = "1.0"
 serde_json = "1.0"
+base64 = "0.13"
+anyhow = "1.0"
+png = "0.17"
```

Comment:
oh dear, the alphabetic sorting is gone; could you restore it?
(oh, btw, i see that you reduced "x.y.z"s to "x.y"s, i like that!)

Reply:
Ohh it was alphabetical, right I'll restore that. This type of ordering is typically my personal preference that I don't usually apply to PRs, not sure why I didn't revert it.